src/main/java/uk/gov/ons/census/casesvc/model/entity/Case.java

```diff
@@ -10,10 +10,16 @@
 import javax.persistence.Id;
 import javax.persistence.OneToMany;
 import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 @Data
 @Entity
+@Builder(toBuilder = true)
```

Comment:
why all the extra annotations? We're going to drown in annotation hell if we annotate everything like this